mods/content/pixel_shifting/pixel_shifting.dm

```diff
@@ -0,0 +1,44 @@
+/mob
+	/// Wether the mob is currently pixel shifted or not
+	var/is_shifted
+	/// If the mob is currently pixel shifting
+	var/shifting
+
+/mob/proc/unpixel_shift()
+	return
+
+/mob/living/unpixel_shift()
+	if(is_shifted)
+		is_shifted = FALSE
+		pixel_x = initial(pixel_x)
+		pixel_y = initial(pixel_y)
+
+/mob/proc/pixel_shift(direction)
+	return
+
+/mob/living/pixel_shift(direction)
+	switch(direction)
+		if(NORTH)
+			if(!canface())
+				return FALSE
+			if(pixel_y <= 16)
```

Comment:
You can replace magic numbers like 16 with define like `DEFAULT_SHIFT_MAX`. 
I use 8 at this moment, but would like to change it to 16 if this visually better.